dependencies.yaml

```diff
@@ -169,6 +169,7 @@ dependencies:
           - h5py>=3.8.0
           - libfaiss>=1.7.1
           - faiss-proc=*=cuda
+          - mathplotlib
```

Comment:
This should be matplotlib? Is `mathplotlib` also a library?

Reply:
Looks like I made a typo